tensorflow/core/graph/graph_test.cc

```diff
@@ -118,6 +118,24 @@ class GraphTest : public ::testing::Test {
     LOG(FATAL) << name;
   }
 
+  bool ControlEdgeExistsInGraphOrNodeDef(const Node* src,
```

Comment:
This logic isn't quite right... you want to test that both the Edge* and the NodeDef input were removed, not either. Maybe divide this into two functions, or change it to "void ExceptControlEdgeRemoved" and it does both checks, or something.

Reply:
This checks if it exists in either the graph or the NodeDef, meaning that it returns `false` only if it's been removed from both. Isn't that what we want?

Reply:
Ah right, my bad.